packages/contracts-bedrock/src/L2/interfaces/ISuperchainERC20Bridge.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
```suggestion
pragma solidity =0.8.25;
```

Reply:
We are leaving interfaces pragmas open to facilitate integrations